RELEASE_NOTES.md

```diff
@@ -1,7 +1,10 @@
+<!-- markdownlint-disable MD024-->
```

Comment:
MD024 is already disabled in the markdownlint file though. If this file is the reason for that, we could also just set the `allow_different_nesting` option to true.

Reply:
Good catch, didn't know about that option. That is exactly what we want and is also present in other files.